client/pages/settings/workspace.vue

```diff
@@ -58,6 +58,69 @@
         </div>
       </div>
 
+      <div class="mt-2">
+        <h4 class="font-bold">Members</h4>
+        <form v-if="isWorkspaceAdmin" @submit.prevent="addUser" class="">
+          <div class="flex flex-col gap-2 mt-2">
```

Comment:
For the front-end form part, please stick to the front-end component we already have: `<input-select>` `<input-text>` etc, which have native support for validation. You can easily find examples on the login page for instance.

Reply:
Also could you please put this create form in a modal triggered by a smaller button? This would de-clutter a bit the UI, since on the cloud version, we already have custom domains on this settings page:
![CleanShot 2024-05-22 at 09 17 30@2x](https://github.com/JhumanJ/OpnForm/assets/11312432/cbd3f58d-b7f8-4594-83b6-b4130019e21e)